config.sh

```diff
@@ -33,7 +50,7 @@ function build_curl_ssl {
     fetch_unpack https://curl.haxx.se/download/curl-${CURL_VERSION}.tar.gz
     (cd curl-${CURL_VERSION} \
         && if [ -z "$IS_OSX" ]; then \
-        LIBS=-ldl ./configure $flags; else \
+        LD_LIBRARY_PATH=$BUILD_PREFIX/lib:$LD_LIBRARY_PATH LIBS=-ldl ./configure $flags; else \
```

Comment:
should't this be `DYLD_LIBRARY_PATH=$BUILD_PREFIX/lib:$DYLD_LIBRARY_PATH` for macOS? This may fix "@rpath/libproj.22.dylib not found" errors found later on. Not sure about the scope, however ...

Reply:
Thanks, I will try it out :+1:

Reply:
Unfortunately, it didn't work.

Reply:
... next guess is `LD_RUNPATH_SEARCH_PATHS` ([docs](https://developer.apple.com/library/archive/documentation/DeveloperTools/Reference/XcodeBuildSettingRef/1-Build_Setting_Reference/build_setting_ref.html#//apple_ref/doc/uid/TP40003931-CH3-SW94)), or other tips described [here](https://blog.krzyzanowskim.com/2018/12/05/rpath-what/). I haven't used a mac for many years, so unfortunately I'm not sure what to believe or test.